src/test/shared/credentials/userCredentialsUtils.test.ts

```diff
@@ -196,20 +195,18 @@ describe('UserCredentialsUtils', () => {
             const result: CredentialsValidationResult = await UserCredentialsUtils.validateCredentials(
                 'fakeaccess',
                 'fakesecret',
-                mockSts as any as AWS.STS)
+                mockSts as any as StsClient)
```

Comment:
Please consider updating these tests to use your new `MockStsClient` class. The mocks will have cleaner declaration, and the `as any as sts` will no longer be necessary.

Reply:
✔️